scripts/apps/vocabularies/views/vocabulary-config-modal.html

```diff
@@ -52,6 +52,13 @@ <h3 class="modal__heading" ng-show="!vocabulary._id"><span translate>Create</spa
                          ng-show="vocabularyForm.$error.maxlength" translate>The helper text must not exceed 120 characters.</div>
                 </div>
 
+                <div class="sd-line-input" ng-if="matchFieldTypeToTab('vocabularies', vocabulary.field_type)">
+                    <label for="cv_selection_type" class="sd-line-input__label" translate>Show to users as:</label>
```

Comment:
`for` attribute only makes sense if there's another element with a corresponding ID. In this case, `select` element should have the id `cv_selection_type`

Reply:
Done

Reply:
Please also fix the issue I mentioned in the review comment